algorithms.tex

```diff
@@ -1,2 +1,17 @@
+% A description of some of the algorithms in SymPy. The list is not
+% exhaustive.
 
+% The sections here are preliminary. We may end up needing to cut some of
+% this.
 
+\subsection{Numerics}
+
+\subsection{Polynomials}
+
+\subsection{The Risch Algorithm}
+
+\subsection{The Gruntz Algorithm}
+
+\subsection{Logic}
+
+\subsection{Other}
```

Comment:
The outline seems pretty good me. But just one thing, instead of mentioning the 'Other' heading as discussed earlier, it would rather be good if we first list all the algorithms, before all the headings,i.e, before **0.1** , and then we can continue describing the topics that have been mentioned in the points **0.1 - 0.5** as of now. It would rather make the structure of the outline better I guess. How about this? Any opinions? :smiley: @asmeurer